scripts/stack_usage/stack_usage.py

```diff
@@ -0,0 +1,671 @@
+                        print_string += '   ' + 'MAX MEMORY USAGE: ' + str(entry[1])
+                        if entry[1] > (.9 * stack_size):
+                            print_string += '   ' + 'WARNING: STACK USAGE EXCEEDS 90% OF STACK SIZE'
+
+            f.write(print_string + '\n')
+            prev_depth = depth
+
+        f.write('\n')
+        f.write('-' * 50 + '\n')
+
+    f.close()
+
+
+# This function prints a help message
+
+# No inputs. Outputs text to the console with information on
+# how to use the script and what the generated output means.
+
+def print_help():
+    print('Usage: python3 syscallsTree.py <output_file.txt> [flags]' + '\n')
+    print('\t' + '<output_file.txt>: Specify the output file name as a .txt file. Ex. output.txt' + '\n')
+    print('Optional flags:' + '\n')
+    print('\t' + '-v: Include full call tree with individual memory usage and informational flags')
+    print('\t' + '-h: Get help message' + '\n')
+    print('Interpreting output:' + '\n')
+    print('\t' + 'Organized by file, each line below the file name represents a function call (if any)' + '\n')
+    print('\t' + 'If verbose option is turned off, output will be the top-most syscall (usually a z_mrsh function) followed by its maximum memory usage according to files generated with CONFIG_STACK_USAGE enabled.')
+    print('\t' + 'If verbose option is turned on, output will include the full call tree on a per-syscall basis with individual memory usage and informational flags.' + '\n')
+    print('\t' + 'Indentation and + symbols indicate depth of function calls. Depth is determined by how the function is linked in the ELF file.')
+    print('\t' + 'A function may be called within an inlined function but because the parent function was inlined, the function call will be linked to the root function who inlined the parent function.')
+    print('\t' + 'As such, the depth of the function call will be under the root function, not its parent function in source code.' + '\n')
+    print('\t' + 'If there are no function calls under a file, it means that the file does not contain any syscalls.' + '\n')
+    print('\t' + 'Memory usage is in bytes. If a function has multiple instances across several object files, the memory usage will be the maximum usage found across all instances.' + '\n')
+    print('Informational flags:' + '\n')
+    print('\t' + 'inlined: Function is inlined')
+    print('\t' + 'pointer: Function is called through a function pointer')
+    print('\t' + 'external: Function is not defined in the same object file')
+    print('\t' + 'static/dynamic/bounded: Flags determined by the .su file generated with -fstack-usage flag when CONFIG_STACK_USAGE is enabled.')
+    print('\t' + '\t' + '\t' + '\t' + 'See https://gcc.gnu.org/onlinedocs/gnat_ugn/Static-Stack-Usage-Analysis.html for more information.')
+    print('\t' + '*: Function was not found in any .su file and has no data on maximum stack usage' + '\n')
```

Comment:
Suggestion: Instead of multiple print statements, can this be a single multi-line print statement?